### PR TITLE
Investigate the categories menu in Arabic

### PR DESCRIPTION
**Pull request by mstriemer**

### CODE REVIEW

**Comment by sherlockedci** [comment]
[ The Adventure of the Naval Build](http://sherlocked.dev.mozaws.net/builds/69350379)

File: sherlocked.js [hunk 1]
```diff
@@ -109,6 +109,19 @@ require('sherlocked')
         .click('mkt-header-child-toggle[for=header--categories]');
 })
 
+.investigate('Category menu on Mobile RTL', function(client) {
```
Comment (ngokevin) [inline sherlocked.js]: Sort of Prefer Caps

**Comment by ngokevin** [comment]
r+

**Comment by sherlockedci** [comment]
[ The Adventure of the Red Build](http://sherlocked.dev.mozaws.net/builds/69358835)

**Pull request merged**